tests/test_herbstluftwm.py

```diff
@@ -13,3 +13,18 @@ def test_herbstluftwm_already_running(hlwm):
                             universal_newlines=True)
     assert result.returncode == 1
     assert re.search(r'another window manager is already running', result.stderr)
+
+
+def test_herbstluftwm_default_autostart(hlwm):
+    expected_tags = [str(tag) for tag in range(1, 10)]
+    # FIXME: Is there a more robust way to find the default autostart path?
+    default_autostart = os.path.join(os.path.abspath(BINDIR), '..', 'share/autostart')
+    autostart = subprocess.Popen(['/usr/bin/env', 'bash', '-e', default_autostart],
+                                 stderr=subprocess.PIPE,
+                                 stdout=subprocess.PIPE,
+                                 universal_newlines=True)
+    autostart.wait(2)
+    assert autostart.returncode == 0
```

Comment:
This can probably be simplified to `subprocess.run(['bash', '-e', default_autostart], check=True)`

Reply:
You're right, your way is so much simpler. Updated the PR accordingly. Thank you!